app/src/main/java/org/simple/clinic/summary/RelativeTimestampGenerator.kt

```diff
@@ -15,17 +15,22 @@ class RelativeTimestampGenerator @Inject constructor() {
   fun generate(today: LocalDateTime, time: Instant): RelativeTimestamp {
```

Comment:
Since this is not being used anywhere except internally and in tests, should we add a `@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)` annotation?

Reply:
Added this myself.